### DISCUSSION TITLE
Azure Active Directory login not working

### DISCUSSION THREAD

**Discussion by camerontbelt** [requarks/wiki#6256] [Error / Bug Report]
Installed wikijs version  2.5.297
I have an IIS site running that forwards to my locally hosted wiki.js instance.
I have followed the documentation here https://docs.requarks.io/auth/azure multiple times to no avail.

Problem 1:

When trying to login with active directory, it redirects me to my open id connect meta endpoint, but instead of login.microsoft.com its my wiki domain. e.g. my.wiki.com/123abc/oauth2/v2.0/authorize.... instead of login.microsoft.com/123abc/oauth2/v2.0/authorize....

Problem 2:

I change the domain to login.microsoft.com, it then redirects successfully to the microsoft login page, I login then it tries to redirect me back to the Redirect URI value found under the configuration reference section.

That gives me the following message.

![Screenshot 2023-03-14 214055](https://user-images.githubusercontent.com/7938124/225191229-7518aa89-7037-45f6-b85d-0d70d185b6a3.png)

Can anyone help me out here?

**Comment by SimonCoutureCSTJ** [comment]
Same type of issue here. Don't know how to fix it...
